src/components/Panel/Channels/index.vue

```diff
@@ -176,6 +180,20 @@ export default {
     },
   },
 
+  watch: {
+    groupChannels (newChannels, oldChannels) {
+      if (newChannels.length > oldChannels.length) {
```

Comment:
I am 99% sure that this won't work due to references -- new and old will be the same. Please doublecheck & test!

Also -- this length difference check might not be good enough. Can cases where a channel is replaced by another channel happen?